INSTALL.md

```diff
@@ -44,6 +48,22 @@ sudo cp libiothsm.so* /usr/lib
 
 Note: if you encountered errors while building the code using OpenSSL 3, you might need to add changes from this [PR](https://github.com/Azure/azure-c-shared-utility/pull/577).
 
+#### 5. Build the EVE specific HSM plugin for Azure IoT Edge 1.4:
+ 
+```bash
+cd azure-on-eve/aziotd
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y
+export PATH="$PATH:/root/.cargo/bin"
+cargo install bindgen --version '^0.54'
+cargo install cbindgen --version '^0.15'
+git clone https://github.com/Azure/iot-identity-service.git
+cp iot-identity-service.diff iot-identity-service/
+cd iot-identity-service
+git reset --hard 15f59c8bd33b1fd8581a74ae6e5ea145c8cb1b9b
+git apply iot-identity-service.diff
+FORCE_NO_UNITTEST=1 make
+```
+
 
 #### 5. After this restart IoT Edge Runtime, for the plugin to take effect:
```

Comment:
Does markdown automatically set the section number? If not, this needs to be changed to 6.

Reply:
fixed